spec/music_album_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rspec'
```

Comment:
- [ ] I'll suggest you remove this `require 'rspec'` statement here. it's throwing an error because you are trying to import `rspec` gem which was not added to the `Gemfile`. Your tests will still run correctly without this. :+1: :100: :smile: